### ISSUE TITLE
Email Template starter/documentation

### ISSUE THREAD

**Issue by ZachRoberts25** [travetto/travetto#40]
https://travetto.dev/docs/email#email-template

A lot of the documentation for where to put the html/scss files is wrong.  Maybe add the email-template piece to the starter as well as update the documentation.  I also need to know how the email-template works

**Comment by arciisine** [comment]
Yeah, the docs are way off. Cleaning them up and will push the out with the 0.7.x release

**Comment by ZachRoberts25** [comment]
the only thing I see missing is where to add css/scss.  I believe the expectation is to add it in a `<style>` tag.  Let me know if thats what your thinking

**Comment by arciisine** [comment]
Sure thing, will add that for the next release (should be this evening).
